slices/stunnel4.yaml

```diff
@@ -0,0 +1,30 @@
+package: stunnel4
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libc6_config
+      - libssl3_libs
+      - openssl_config
+      - libgssapi-krb5-2_libs
```

Comment:
I believe that once the previously mentioned dependency gaps are addressed, this list of essentials can be reduced. For example, `libgssapi-krb5-2_libs` is not a direct dependency of `stunnel4`, but it will come in via `libtirpc3`